### DISCUSSION TITLE
Adding read/unread totals on series list instead of green bar

### DISCUSSION THREAD

**Discussion by DasVekman** [Kareadita/Kavita#2928] [Ideas]
### Idea Description

Ideally, I asked for a way to show how many books are in a series as well as of those what's been read. So if you have read 2 books of 5, it will show 2/5 and if you haven't started it yet, it will show 0/5. This would display in the series tile. I can provide a screenshot mock up if needed. Thank you for your consideration. 

### Idea Category

Web UI

### Duration of Using Kavita

A few months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by hotsno** [comment]
I agree. I think that the green bar is honestly almost completely useless and just adds noise to the UI, since it doesn't tell you what chapter/volume/whatever the progress is for. I'd prefer a (fairly big and/or readable) unread count badge placed on the manga cover, and/or just have that information written in text somewhere like OP suggests.

**Comment by therobbiedavis** [comment]
@DasVekman @hotsno We have a major UI revamp in the works and I planned on making the progress bar info a bit more visible by showing the percentage, however it seems like you want to see precisely how much is left. That's difficult because you then need to specify which number you are calculating from. I believe by default we use chapter/issue because that provides the most accurate number since volumes are usually retroactive, especially for mangas. 
Are you wanting to display `Issues Read / Issues Total` or `Volumes Read / Volumes Total`?

Mock up of the UI redesign:
![image](https://github.com/Kareadita/Kavita/assets/7907265/4fc9a520-e704-444b-ae0e-9b5ab3fdbfae)

**Comment by DasVekman** [comment]
I'd like it to be visible from the series list. For books, even if it's just like I have a series with 6 books and I've read 2, it would show 2/6
________________________________

**Reply by rix8675** [reply]
this right here would be amazing.
two methods this could be added.
 - just like in newly added(kinda like how plex does) i'd just use the unread count as the 2/6 would cover more of the image.
 - using the purple bar(making it slightly thicker) you could add the 2/6 as it'd be more easily read.
